code/game/objects/items/weapons/weaponry.dm

```diff
@@ -10,6 +10,7 @@
 	throw_range = 4
 	throwforce = 10
 	w_class = 2
+	var/global/list/nullchoices

```

Comment:
Why does that need to be a global var?

Reply:
This is a modified version of the code used for chameleon items. So, I guess for the same reason it's global there? Does it need to be changed?

Reply:
not really, it only needs to be global if some other things in other file is trying access it. But then they can just access var from the object itself. So I see no reason for it to be global

Reply:
There is no reason to make each instance regenerate this list as it's always the same. This is global so it only gets generated once.

Reply:
Wouldn't it be better noted as `static` then? Granted, in DM the difference might well be semantics. But the behaviour of this matches more closely what the classical definition for a class static variable is.

Reply:
`static` would probably be clearer, yes.